bundles/org.openhab.ui/web/src/assets/definitions/metadata/namespaces.js

```diff
@@ -1,5 +1,6 @@
 export default [
   { name: 'stateDescription', label: 'State Description' },
+  { name: 'unit', label: 'Internal Unit' },
```

Comment:
@J-N-K WDYT about the labelling?

Reply:
I think it could be labelled "Measurement Unit" or even "Unit" and promoted to the top (stateDescription and commandDescription are somehow linked so it feels wrong to list a metadata namespace between them).

Reply:
Partly agreed, let's put it to the top.
What I don't like about "Unit" or "Measurement Unit" that they don't make clear that it is a internal unit. The user could think that they are for display purpose (even though the description says they aren't, but I'm not sure if every user always reads those descriptions carefully).

Reply:
> they don't make clear that it is a internal unit

Yet I (and I suppose many users who will make decisions based on these descriptions) am not sure what an "internal" unit is.

Reply:
If no state description is set (which is the default), it is also used for displaying the values.